SparkConnector/setup.py

```diff
@@ -71,7 +71,6 @@
     cmdclass= cmdclass,
     packages=setuptools.find_packages(),
     install_requires=[
-        "jupyterlab>=4.0.0,<5",
```

Comment:
can you still keep it commented out?

Reply:
@diocas, we are commenting out this dependency because when installing sparkconnector in a custom environment, it pulls jupyterlab and a lot of dependencies with it, but we do not need all that to be installed in the environment (we tried). Do you have an objection to this?

Reply:
Discuss privately: we might be able to get rid of bs4 too.